<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_continuar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CONTINUAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dato_ip">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dato_puerto">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dato_nombre">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_nombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NOMBRE :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_puerto">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PUERTO : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ip">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TU IP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-30</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>37</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WOLFESTEIN</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>591</width>
      <height>201</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>575</width>
       <height>199</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_partidas">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Partidas Disponibles :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
